### PR TITLE
Update Exception.php

### PR DESCRIPTION
**Pull request by tmihalik**

### CODE REVIEW

**Comment by tmihalik** [comment]
Hallo!!!

**Comment by tmihalik** [comment]
alt text

**Comment by evanpurkhiser** [comment]
Yep! Don't need to be in any rush here though, it doesn't look like a criticial issue =)

**Comment by tmihalik** [comment]
Not critical... but it's a very very simple typo.

**Comment by zeelot** [comment]
FINE! :)

**Comment by tmihalik** [comment]
THANK YOU!!!!

File: classes/Kohana/HTTP/Exception.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ public function __construct($message = NULL, array $variables = NULL, Exception
 	 * Store the Request that triggered this exception.
 	 *
 	 * @param   Request   $request  Request object that triggered this exception.
-	 * @return  Response
+	 * @return  HTTP_Exception
```
Comment (sergeyklay) [inline classes/Kohana/HTTP/Exception.php:49]: `@return  HTTP_Exception|Request`

**Pull request merged**